include/ignition/gazebo/rendering/SceneManager.hh

```diff
@@ -214,6 +214,14 @@ inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
         std::vector<common::TrajectoryInfo>& _trajectories,
         std::chrono::steady_clock::time_point _time);
 
+    /// \brief Load Actor trajectories
+    /// \param[in] _actor Actor
+    /// \return Trajectory vector
+    public: std::vector<common::TrajectoryInfo>
+        LoadTrajectories(const sdf::Actor &_actor,
+        std::unordered_map<std::string, unsigned int> &_mapAnimNameId,
+        Entity &_id);
```

Comment:
Can this be a private function inside `SceneManagerPrivate`? This header is installed, so whatever we add here will be hard to change in the future. But whatever is in the `.cc` file can be changed at any time!

Reply:
Done.